ps_emailalerts.php

```diff
@@ -123,13 +123,16 @@ public function install($delete_params = true)
 
             $sql = 'CREATE TABLE IF NOT EXISTS `'._DB_PREFIX_.MailAlert::$definition['table'].'`
 				(
+					`id_mailalert` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
 					`id_customer` int(10) unsigned NOT NULL,
 					`customer_email` varchar(128) NOT NULL,
 					`id_product` int(10) unsigned NOT NULL,
 					`id_product_attribute` int(10) unsigned NOT NULL,
 					`id_shop` int(10) unsigned NOT NULL,
 					`id_lang` int(10) unsigned NOT NULL,
-					PRIMARY KEY  (`id_customer`,`customer_email`,`id_product`,`id_product_attribute`,`id_shop`)
+                    `date_add` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
weird indentation :)

Reply:
i now, but why showing like this?
In my github desktop and Visual Studio shows ok. 

![2020-01-27_16h00_48](https://user-images.githubusercontent.com/4552670/73185353-33acac00-411e-11ea-886b-15144d973766.png)
![2020-01-27_16h01_43](https://user-images.githubusercontent.com/4552670/73185424-50e17a80-411e-11ea-9f27-91c0ffd8cb8d.png)

Reply:
it's because original lines are tabs not spaces